src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    const inversedRobotValue = robot[key];
+
+    if (inversedRobot.hasOwnProperty(robot[key])) {
+      return null;
+    }
+
+    inversedRobot[inversedRobotValue] = key;
```

Comment:
You can either remove `inversedRobotValue` or use it everywhere like this:

```suggestion
    const inversedRobotValue = robot[key];

    if (inversedRobot.hasOwnProperty(inversedRobotValue) {
      return null;
    }

    inversedRobot[inversedRobotValue] = key;
```